Sources/Kitura/Router.swift

```diff
@@ -126,14 +126,22 @@ public class Router {
 
     // MARK: Template Engine
 
-    /// Register a templating engine. The templating engine will handle files in the `viewsPath`
-    /// that match the file extension it supports.
+    /// A template engine allows rendering of documents using static templates.
+    ///
+    /// By default the templating engine will handle files in the `./Views` directory
+    /// that match the file extension it supports. You can change this default location using the `viewsPath` property.
     /// ### Usage Example: ###
     /// ```swift
     /// let router = Router()
+    /// //Register a template engine to a given router instance
     /// router.add(templateEngine: MyTemplateEngine())
+    /// //Register a template engine to a given router instance using a custom file extension
     /// router.add(templateEngine: MyOtherTemplateEngine(), forFileExtensions: ["html"], useDefaultFileExtension: false)
+    /// //Change the location a template engine will look for template files in any given router instance
+    /// router.viewsPath = "New_Custom_Location"
```

Comment:
This doesn't need to be here, it's already been covered in https://ibm-swift.github.io/Kitura/Classes/Router.html#/s:6Kitura6RouterC9viewsPathSSv